plugin/storage/es/spanstore/dbmodel/to_domain.go

```diff
@@ -126,10 +126,40 @@ func (td toDomain) convertKeyValues(tags []json.KeyValue) ([]model.KeyValue, err
 	return retMe, nil
 }
 
+func addTagsMap(tagsMap map[string]interface{}) []model.KeyValue {
+	kvs := make([]model.KeyValue, len(tagsMap))
+	i := 0
+	for k, v := range tagsMap {
+		dKey := strings.Replace(k, ":", ".", -1)
+		tag := model.KeyValue{Key: dKey}
+		// TODO v.(int/64/32) didn't work it is always a float
+		if pInt, err := strconv.ParseInt(fmt.Sprintf("%v", v), 10, 64); err == nil {
```

Comment:
ES always returns float64 as a number - even if it was stored as int. This works but it's not that pretty.